### DISCUSSION TITLE
域名备案

### DISCUSSION THREAD

**Discussion by kmg76** [natfrp/wiki#114] [水深火热]
腾讯云买的域名，备案必须得买腾讯云服务器，没备案访问放出<<白名单设备自动拦截>>

**Comment by mhzx** [comment]
没有服务码没办法备案，没备案又没办法使用，好难啊

**Comment by mhzx** [comment]
服务码又得买服务器，有服务器就不需要穿透了，似乎无解

**Reply by fakeboboliu** [reply]
请不要用无关的牢骚堆积我们的邮箱，这会让我们无法及时处理其他问题

**Comment by tidymoon** [comment]
他甚至整了个小号来骂 他真的 我哭死（
话说回来 备案问题请自行解决 若不想备案可以使用国外服务器啊 只是速度嘛...

**Reply by mhzx** [reply]
这不是想不想备案的问题，这是有没有办法备案的问题。用别的服务器服务码即使备案了，解析过来过段时间也会掉，因为你解析的位置和实际服务器不同

**Reply by Lurito** [reply]
> 这不是想不想备案的问题，这是有没有办法备案的问题。用别的服务器服务码即使备案了，解析过来过段时间也会掉，因为你解析的位置和实际服务器不同

你在阿里云或腾讯云等服务商按正规渠道备案，用根域名或者www域名绑一个对象存储的存储桶，里面挂一个静态页面写上备案号，开启静态网站，这样 100% 合规。然后你就可以用子域名绑 FRP 服务了。
